toolchain/sem_ir/name_scope.h

```diff
@@ -81,6 +81,11 @@ struct NameScope : Printable<NameScope> {
                  name_entry.name_id);
   }
 
+  // Returns true if this name scope describes a local or imported package.
+  auto IsImportedPackage() const -> bool {
```

Comment:
```suggestion
  auto is_imported_package() const -> bool {
```

FWIW, might consider using accessor naming to reflect that the implementation is trivial.